pymatgen/io/vasp/incar_parameters.json

```diff
@@ -1,653 +1,1447 @@
+     ]
+ },
+ "ISYM": {
+   "type": "int",
+   "values": [
+      -1,
+      0,
+      1,
+      2,
+      3
+     ]
+ },
+ "IVDW": {
+   "type": "int",
+   "values": [
+      0,
+      1,
+      10,
+      11,
+      12,
+      2,
+      20,
+      21,
+      202,
+      4
+     ]
+ },
+ "IWAVPR": {
+   "type": "int",
+   "values": [
+      0,
+      1,
+      2,
+      3,
+      10,
+      11,
+      12,
+      13
+     ]
+ },
+ "KBLOCK": {
+    "type": "int"
+ },
+ "KGAMMA": {
+    "type": "bool"
+ },
+ "KPAR": {
+    "type": "int"
+ },
+ "KPOINT_BSE": {
+    "type": "list"
+ },
+ "KPUSE": {
+    "type": "list"
+ },
+ "KSPACING": {
+    "type": "float"
+ },
+ "LADDER": {
+    "type": "bool"
+ },
+ "LAECHG": {
+    "type": "bool"
+ },
+ "LAMBDA": {
+    "type": "float"
+ },
+ "LANGEVIN_GAMMA": {
+    "type": "list"
+ },
+ "LANGEVIN_GAMMA_L": {
+    "type": "float"
+ },
+ "LASPH": {
+    "type": "bool"
+ },
+ "LASYNC": {
+    "type": "bool"
+ },
+ "LATTICE_CONSTRAINTS": {
+    "type": "list"
+ },
+ "LBERRY": {
+    "type": "bool"
+ },
+ "LBLUEOUT": {
+    "type": "bool"
+ },
+ "LBONE": {
+    "type": "bool"
+ },
+ "LCALCEPS": {
+    "type": "bool"
+ },
+ "LCALCPOL": {
+    "type": "bool"
+ },
+ "LCHARG": {
+    "type": "bool"
+ },
+ "LCHIMAG": {
+    "type": "bool"
+ },
+ "LCORR": {
+    "type": "bool"
+ },
+ "LDAU": {
+    "type": "bool"
+ },
+ "LDAUJ": {
+    "type": "list"
+ },
+ "LDAUL": {
+    "type": "list"
+ },
+ "LDAUPRINT": {
+   "type": "int",
+   "values": [
+      0,
+      1,
+      2
+     ]
+ },
+ "LDAUTYPE": {
+   "type": "int",
+   "values": [
+      1,
+      2,
+      4
+     ]
+ },
+ "LDAUU": {
+    "type": "list"
+ },
+ "LDIAG": {
+    "type": "bool"
+ },
+ "LDIPOL": {
+    "type": "bool"
+ },
+ "LEFG": {
+    "type": "bool"
+ },
+ "LELF": {
+    "type": "bool"
+ },
+ "LEPSILON": {
+    "type": "bool"
+ },
+ "LFOCKAEDFT": {
+    "type": "bool"
+ },
+ "LHARTREE": {
+    "type": "bool"
+ },
+ "LHFCALC": {
+    "type": "bool"
+ },
+ "LHYPERFINE": {
+    "type": "bool"
+ },
+ "LKPROJ": {
+    "type": "bool"
+ },
+ "LLRAUG": {
+    "type": "bool"
+ },
+ "LMAXFOCK": {
+    "type": "int"
+ },
+ "LMAXFOCKAE": {
+    "type": "int"
+ },
+ "LMAXMIX": {
+    "type": "int"
+ },
+ "LMAXPAW": {
+    "type": "int"
+ },
+ "LMAXTAU": {
+    "type": "int"
+ },
+ "LMIXTAU": {
+    "type": "bool"
+ },
+ "LMONO": {
+    "type": "bool"
+ },
+ "LNABLA": {
+    "type": "bool"
+ },
+ "LNMR_SYM_RED": {
+    "type": "bool"
+ },
+ "LNONCOLLINEAR": {
+    "type": "bool"
+ },
+ "LOCPROJ": {
+    "type": "list"
+ },
+ "LOPTICS": {
+    "type": "bool"
+ },
+ "LORBIT": {
+   "type": "int",
+   "values": [
+      0,
+      1,
+      2,
+      5,
+      10,
+      11,
+      12
+     ]
+ },
+ "LORBMOM": {
+    "type": "bool"
+ },
+ "LPARD": {
+    "type": "bool"
+ },
+ "LPEAD": {
+    "type": "bool"
+ },
+ "LPLANE": {
+    "type": "bool"
+ },
+ "LREAL": {
+   "values": [
+      false,
+      true,
+      "O",
+      "A",
+      "Auto",
+      "On"
+     ]
```

Comment:
I'm still looking into possible way to record more complex type hints like `bool | str`